index.rst

```diff
@@ -58,7 +58,7 @@ instructions please see the :ref:`setup guide <setup>`.
    not require any issue to be created.
 
 6. Once you fixed the issue, run the tests, run ``make patchcheck``, and if
-   everything is ok, commit.
+   everything is ok, commit. On windows run ``.\python.bat Tools\patchcheck\patchcheck.py``
```

Comment:
Perhaps something like:

```rst
6. Once you fixed the issue, run the tests,
   run ``make patchcheck`` (Unix/macOS)
   or ``.\python.bat Tools\patchcheck\patchcheck.py`` (Windows),
   and if everything is ok, commit.
```

I'm using macOS; someone on Windows will have to confirm if this command works.

Reply:
Yeah, this is clearer. I'm probably gonna modify it tomorrow.

I got the command from [here](https://devguide.python.org/getting-started/pull-request-lifecycle/#patchcheck), but I'm guessing it's still necessary to have someone confirm it.

Reply:
We should also use Furo's tab feature to display the commands in macOS / windows.
https://pradyunsg.me/furo/reference/tabs/


e.g. 


```
.. tab:: macOS

    make patchcheck

.. tab:: Windows

    .\python.bat Tools\patchcheck\patchcheck.py 

```